sketch/hash_ordered.hpp

```diff
@@ -87,12 +87,9 @@ class OrderedMinHash : public HashBase<T> {
      * @tparam C the type of characters in the sequence
      */
     template <typename C>
-    Vec2D<T> compute(const std::vector<C> &sequence, uint32_t k, uint32_t alphabet_size) {
-        Timer::start("compute_sequence");
+    void compute(Vec2D<T> &sketch, const std::vector<C> &sequence, uint32_t k, uint32_t alphabet_size) {
         std::vector<T> kmers = seq2kmer<C, T>(sequence, k, alphabet_size);
-        Vec2D<T> sketch = compute(kmers);
-        Timer::stop();
-        return sketch;
+        sketch = compute(kmers);
```

Comment:
Just return the value rather than sending it as a parameter by reference. RVO will make sure that this is done efficiently.